pages/02.administer/10.install/20.dns_config/dns_config.md

```diff
@@ -91,7 +91,7 @@ Though it might be easier to understand it if displayed like this:
 - These are example values ! See your generated conf for the actual values you should use;
 - We recommend a [TTL](https://en.wikipedia.org/wiki/Time_to_live#DNS_records) of 3600 (1 hour). But you can use something else if you know what you're doing;
 - Don't put an IPv6 record if you're not sure IPv6 really works on your server! You might have issues with Let's Encrypt if it doesn't.
-- If you're using the domain provider Namecheap the SRV DNS entries are formatted as **Service**: _xmpp-client **Protocol**: _tcp **Priority**: 0 **Weight**: 5 **Port**: 5222 **Target**: your.domain.tld
+- If you're using the domain provider Namecheap the SRV DNS entries are formatted as **Service**: _xmpp-client **Protocol**:_tcp **Priority**: 0 **Weight**: 5 **Port**: 5222 **Target**: your.domain.tld
```

Comment:
hmm, that one looks like a legit space, otherwise `_foobar_` with be interpreted as bold/italic ? But it looks like we should have some backticks in there because that's pretty much code

Reply:
https://github.com/DavidAnson/markdownlint/blob/v0.33.0/doc/md037.md 🤔